scripts/upstream.sh

```diff
@@ -59,12 +59,12 @@ message_rebase_dirty() {
 cat <<EOF
 
 The patch set did not apply cleanly. You need to manually resolve
-rebase conflicts directly in './upstream'. When you have completed
-the rebase, run
+rebase conflicts directly in './upstream' and run 'git rebase --continue'.
+When you have completed the rebase, run
```

Comment:
You might not want to run `git rebase --continue`. There are multiple relevant rebase commands (`git rebase --skip` comes to mind in this situation).

I understand wanting to add emphasis here, but we should be careful to not be ovecrperscriptive.

```suggestion
rebase conflicts directly in './upstream' finish the rebase.
In the simple case, this could mean running 'git rebase --continue'.
When you have completed the rebase, run
```